features/account/customer_account/address_book/viewing_addresses.feature

```diff
@@ -6,15 +6,15 @@ Feature: Viewing my address book
 
     Background:
         Given the store operates on a single channel in "United States"
-        And I am a logged in customer
+        And I am a logged in customer with name "Lucifer Morningstar"
```

Comment:
I changed it, because IMO scenario `Inability to view the addresses of other customers` is not clearly and which is customer about in step `this address should be assigned to "Lucifer Morningstar"`, we have added address with this name for this customer, but this customer is not ` "Lucifer Morningstar"` but random named